src/main/scala/outwatch/dom/DomTypes.scala

```diff
@@ -92,6 +92,8 @@ trait ReflectedAttrs
   extends reflectedAttrs.ReflectedAttrs[CodecBuilder.Attribute]
   with ReflectedAttrBuilder[CodecBuilder.Attribute] {
 
+  override lazy val className: AccumAttributeBuilder[String] = new AccumAttributeBuilder("class", _.toString, _ + " " + _)
```

Comment:
We also need to overwrite cls from domtypes!

Reply:
You are correct, I was just noticing that. I declared `AttributesCompat` and `TagsCompat` self types to `Attributes` and `Tags`, which should take care of that I think.

However, there's still a compilation issue on 2.11, do you have any idea why? Seems to only happen when importing `className` form the `dom` package object, using `Attributes.className` works fine.

Reply:
Hmm, I have no idea. It is really weird working in the dom package, because it has all these properties automagically. But it should have worked as it was before? We can clean it up with #124.